--- DA14580_581_SDK_3.0.8.0/dk_apps/src/ip/ble/ll/src/rwble/rwble.c	2015-03-11 13:56:20.000000000
+++ src/src/ip/ble/ll/src/rwble/rwble.c	2015-10-01 01:18:20.000000000
@@ -565,13 +565,13 @@
 #endif //if 0
 
 /*********************************************************************************
  *** RX_INT ISR
  ***/
 #ifdef __GNUC__
-void __wrap_BLE_RX_Handler(void)
+void BLE_RX_Handler(void)
 #else
 void $Sub$$BLE_RX_Handler(void)
 #endif
 {
     
 #ifdef METRICS
@@ -582,13 +582,13 @@
 }
 
 /*********************************************************************************
  *** EVENT_INT ISR
  ***/
 #ifdef __GNUC__
-void __wrap_BLE_EVENT_Handler(void)
+void BLE_EVENT_Handler(void)
 #else
 void $Sub$$BLE_EVENT_Handler(void)
 #endif
 {
     // Check BLE interrupt status and call the appropriate handlers
     //uint32_t irq_stat = ble_intstat_get();
@@ -991,13 +991,13 @@
 }
 
 #endif // USE_POWER_OPTIMIZATIONS
 
 
 #ifdef __GNUC__
-void __wrap_BLE_RF_DIAG_Handler(void)
+void BLE_RF_DIAG_Handler(void)
 #else
 void $Sub$$BLE_RF_DIAG_Handler(void)
 #endif
 {
     uint32 irq_scr;
     uint16_t cn;    
